Android App/app/src/main/java/org/cfp/citizenconnect/Data/FragmentDataSet.java

```diff
@@ -52,7 +52,7 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         progressDialog.setCancelable(false);
         mGridView = rootView.findViewById(R.id.dataSet_GridView);
         mList = new ArrayList<>();
-        getDataSetLayout(database.getReference(DATASET_REFFERENCE), response -> {
+        getLayout("DATASET",database.getReference(DATASET_REFFERENCE), response -> {
```

Comment:
better define `DATASET` as `static final String`

Reply:
agreed

Reply:
but ignoring for now